includes/templates/template_default/sideboxes/tpl_featured_categories.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * Side Box Template
+ *
+ * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: DrByte 2024 May 24 Modified in v2.1.0-alpha1 $
+ * based on tpl_featured
+ */
+$content = "";
+$content .= '<div class="sideBoxContent centeredContent">';
+$featured_category_box_counter = 0;
+while (!$random_featured_categories->EOF) {
+    $data = (new Category((int)$random_featured_categories->fields['categories_id']))->withDefaultLanguage()->getData();
+    $featured_category_box_counter++;
+    $content .= "\n" . '  <div class="sideBoxContentItem">';
+    $content .= '<a href="' . zen_href_link(FILENAME_DEFAULT, 'cPath='.  zen_get_generated_category_path_rev($data["categories_id"])). '">'
```

Comment:
- Use single-, not double-, quotes around the `categories_id`.
- Add single space before the `.` concatenation operator; remove extra space after.

Reply:
Still missing the single-space before the various concatenation operators.